pkg/squashfs/squashfs.go

```diff
@@ -135,7 +155,6 @@ func (s *Snap) Build(buildDir string) error {
 		return runCommand(
 			"mksquashfs",
 			".", fullSnapPath,
-			"-all-root",
```

Comment:
ftr, i mean: a comment by you in the pull request would be appropriate here, not in the code itself